.github/workflows/bench.yml

```
@@ -0,0 +1,37 @@
+name: 'Run benchmark'
+
+on: # rebuild any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

let's commit this right before merge it

```suggestion
    runs-on: benchmarking-runner
```